Solutions/Corvus.Testing.AzureFunctions/Corvus/Testing/AzureFunctions/FunctionStartupException.cs

```diff
@@ -0,0 +1,29 @@
+﻿// <copyright file="FunctionStartupException.cs" company="Endjin Limited">
+// Copyright (c) Endjin Limited. All rights reserved.
+// </copyright>
+
+namespace Corvus.Testing.AzureFunctions
+{
+    using System;
+
+    /// <inheritdoc />
```

Comment:
Please add a comment describing the purpose of this class. We generally use `<inheritdoc/>` only for interface implementations and overrides, where the inherited documentation already completely describes the method.